dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/LabelMapperTest.java

```diff
@@ -47,10 +47,65 @@ class LabelMapperTest {
 
   public static final String ENROLLMENT_DATE = "Enrollment date";
 
-  private static final String NAME_SCHEDULED_DATE = "Scheduled date";
+  private static final String SCHEDULED_DATE = "Scheduled date";
+
+  private static final String ORG_UNIT = "Organisation unit";
+
+  private static final String PROGRAM_STAGE = "Program stage";
+
+  private static final String ENROLLMENT = "Enrollment";
+
+  private static final String EVENT = "Event";
+
+  @Test
+  void testGetHeaderNameFor_EVENT() {
```

Comment:
Use standard camel-case for method names (`_event`).

Reply:
The idea here is to give a better context and make it clear that these tests are directly related to the CONSTANTS. In this case `EVENT`.

I can make it lowercase, but it will reduce the readability and at first sight, it will give the impression that this is testing an Event object, instead...